src/frontend/components/UI/Sidebar/components/SidebarLinks/index.tsx

```diff
@@ -70,13 +70,15 @@ export default function SidebarLinks() {
       setSettingsPath('/settings/app/default/general')
     } else {
       getGameInfo(appName, runner).then((info) => {
-        setGameInfo(info)
-        if (info?.is_installed) {
-          setIsDefaultSetting(false)
-          const wineOrOther = isWin
-            ? `/settings/${runner}/${appName}/other`
-            : `/settings/${runner}/${appName}/wine`
-          setSettingsPath(wineOrOther)
+        if (info) {
+          setGameInfo(info)
+          if (info?.is_installed) {
```

Comment:
I think the `?` is not needed now?

Reply:
There are probably all kinds of optional chaining and non-null assertion operators in the frontend that can now be removed

There's [an eslint rule](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/no-unnecessary-condition.md) that makes finding these easy, I'll give that a go

Reply:
Uh oh, that was a bad idea, >200 problems with that rule

It might be better to make those changes in a separate PR then

Reply:
Got them all solved now, but it's ~600 lines of changes, so I'll add those in another PR